### DISCUSSION TITLE
Prevent registration-process for new users when disabled

### DISCUSSION THREAD

**Discussion by kangaroo72** [dani-garcia/vaultwarden#3393] [Ideas]
Hi,

isn't it possible, to inform the user, that it's not allowed to register if disabled in admin-panel?

I have disabled the registration of new users in admin-panel.
Anyway it's still possible to enter the data. The information, that it's disabled will popup after submitting the registration.
I think, it makes sense to directly inform, that it's disabled.
What do you think?

**Comment by BlackDex** [comment]
This has been discussed a lot of times. But that isn't possible.
The web-vault does not support this. Also, you can disable signups, but still allow invites. They still need to go via the same interface. No way to determine up front what to show the user, since there is no call done to the server at that point., So the interface is still needed.

**Comment by kangaroo72** [comment]
Thank you!!
